fe/fe-core/src/main/java/com/starrocks/sql/optimizer/MvPlanContextBuilder.java

```diff
@@ -19,10 +19,15 @@
 import com.starrocks.qe.ConnectContext;
 
 public class MvPlanContextBuilder {
-    public MvPlanContext getPlanContext(MaterializedView mv) {
+    public static MvPlanContext getPlanContext(MaterializedView mv) {
         // build mv query logical plan
         MaterializedViewOptimizer mvOptimizer = new MaterializedViewOptimizer();
         ConnectContext connectContext = new ConnectContext();
+        // If the caller is not from query (eg. background schema change thread), set thread local info to avoid
```

Comment:
remove line 26, and use ConnectContext.get() in line 33 directly?
I have came to an npe here and fix it by the above method temperarily

Reply:
From `SchemaChangJobV2`, there is no connectContext in the thread local, but in the optimizer `ConnectContect.get()` is used in many places, need to set it into the thread local to avoid NPE.